### ISSUE TITLE
Update site not working

### ISSUE THREAD

**Issue by daniloqueiroz** [mystilleef/eclipse4-fastopen#4]
The current version on market place doesn't work on Eclipse Mars and the update site is returning 404.

**Comment by daniloqueiroz** [comment]
Ok, I've found the update site, but anyway seems that the version there is outdated (20150519) and don't include the fix for eclipse mars (https://github.com/mystilleef/eclipse4-fastopen/commit/3312ac38cddad4be00ca944e89174ca69521f7d5).
Also the docs point to the wrong update site, it should be https://raw.githubusercontent.com/mystilleef/eclipse4-fastopen-updatesite/master/com.laboki.eclipse.updatesite.fastopen/site.xml .

**Comment by mikehopemar** [comment]
Exactly. Version 20150519 don't include the fix in luna and mars. I have an empty window to choose a files. File phisically can be opened by touching arrow keys, but view is empty.
